src/Time/StepChoosers/CFL.hpp

```diff
@@ -0,0 +1,90 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <limits>
+#include <pup.h>
+
+#include "Domain/MinimumGridSpacing.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Parallel/ConstGlobalCache.hpp"
+#include "Time/StepChoosers/StepChooser.hpp"  // IWYU pragma: keep
+#include "Time/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace Parallel {
+template <typename Metavariables>
+class ConstGlobalCache;
+}  // namespace Parallel
+/// \endcond
+
+namespace StepChoosers {
+
+/// Suggests a step size based on the CFL stability criterion.
+template <size_t Dim, typename Frame, typename StepChooserRegistrars>
+class CFL : public StepChooser<StepChooserRegistrars> {
```

Comment:
`Cfl` would be camel case

Reply:
I don't think we've actually had a discussion about this.  Do we have a naming convention for input file options?

Reply:
We discussed it and decided camel case